xml_inscriptions/transcribed/CA.Malibu.JPGM.G.71.AA.121.xml

```diff
@@ -64,9 +65,16 @@
                   </summary>
                   <origin>
                      <!--Should contain information about origin of inscription including place and date.-->
-                     <date evidence="unspecified" notBefore="-0360" notAfter="-0360">360 BCE</date>
+                     <date evidence="unspecified" notBefore="-0360" notAfter="-0380">c. 370
+                        BCE</date>
                      <placeName ref="europe.greece.attica">Greece, Attica</placeName>
                   </origin>
+                  <provenance type="observed" subtype="recorded">
+                     <p>In the hands of <persName role="collector">Nicolas Koutoulakis</persName> by <date notAfter="1971">1970-1971</date>.</p>
+                  </provenance>
+                  <provenance type="transferred" subtype="purchased">
+                     <p>Purchased by the <orgName role="buyer collector">J. Paul Getty Museum</orgName> in <date when="1971">1971</date> by <persName role="seller">Nicolas Koutoulakis</persName></p>
```

Comment:
*from* Koutoutalkis, not by? Or the `@role` is wrong in his second appearance here